server/sonar-web/src/main/js/helpers/periods.js

```diff
@@ -51,7 +50,7 @@ export function getPeriodDate(period) {
     return null;
   }
 
-  return moment(period.date).toDate();
+  return new Date(period.date);
```

Comment:
Here and everywhere: safari doesn't parse dates in the format they're sent by our server, that's why `moment(...).toDate()` was used. Now without moment I'm afraid we have to write our own parsing function 😢